development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
```

Comment:
Yeah, we'd just also have to override the actual age_group column in the Attendee table to use a foreign ID instead of a dict.